apps/explorer/src/ui/InternalLink.tsx

```diff
@@ -16,17 +18,16 @@ function createInternalLink<T extends string>(
 ) {
     return ({
         [propName]: id,
+        moduleId,
```

Comment:
Instead of putting this here, just extend the props when creating the module link.

Reply:
Alternatively we could just add a `query` prop to the link to do this more generically but def shouldn't do it with moduleId special casing.